src/robusta/core/reporting/base.py

```diff
@@ -187,11 +189,32 @@ def attribute_map(self) -> Dict[str, str]:
             "name": str(self.subject.name),
         }
 
+    def _map_service_to_uri(self):
+        if not self.service:
+            return "graphs"
+        if self.service.resource_type.lower() == "job":
+            return "jobs"
+        return "services"
+
+    def get_investigate_uri(self, account_id: str, cluster_name: Optional[str] = None):
+        uri_path = self._map_service_to_uri()
+        params = {
+            "account": account_id,
+            "clusters": f"[\"{cluster_name}\"]" if cluster_name else None,
```

Comment:
if it's always a single cluster, maybe it's better to call it `cluster` and `namespace` ?

Reply:
That’s how parameters are passed on the frontend in the query string, that’s why it is in plural

Reply:
got it